packages/db-objects-file/lib/objects/objectsInMemFileDB.js

```diff
@@ -2075,11 +2065,12 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             options.acl = null;
         }
 
-        utils.checkObjectRights(this, id, this.dataset[id], options, utils.CONSTS.ACCESS_WRITE, (err, options) => {
```

Comment:
If we use the options argument from `checkObjectRights` we will always end up having admin/admin if no user/group provided. Then the code will later overwrite the default user/group with the user/group attribute which will result in ignoring `system.config` default acl user/group